### PR TITLE
Secret watcher run on startup

### PR DESCRIPTION
**Pull request by rafael-azevedo**
This PR has secretwatcher run on startup and logs if there are any errors. Once that is complete secretwatcher runs as normal every 10 minutes

### CODE REVIEW

File: pkg/credentialwatcher/secretwatcher.go [hunk 1]
```diff
@@ -52,6 +52,12 @@ func NewSecretWatcher(client client.Client, watchInterval time.Duration) *secret
 // message is sent on the stopCh
 func (s *secretWatcher) Start(log logr.Logger, stopCh <-chan struct{}) {
 	log.Info("Starting the secretWatcher")
+	log.Info("Secretwatcher initail run")
```
Comment (fahlmant) [inline pkg/credentialwatcher/secretwatcher.go]:
```suggestion
	log.Info("Secretwatcher initial run")
```

**Comment by rogbas** [comment]
@ArielLima please can you test that PR, confirm the operator starts ok with it?

**Comment by ArielLima** [comment]
I tested this and did not get an error or output (I assume it still worked):
```
{"level":"info","ts":1584711433.826213,"logger":"cmd","msg":"Starting the secretWatcher"}
{"level":"info","ts":1584711433.826324,"logger":"cmd","msg":"Secretwatcher initial run"}
{"level":"info","ts":1584711433.85499,"logger":"cmd","msg":"Secretwatcher initial run complete"}
```
last line in the log was inserted simply for testing.

**Comment by ArielLima** [comment]
@fahlmant Question on this. The expected result is no output?

**Pull request merged**